integreat_cms/core/settings.py

```diff
@@ -114,9 +114,13 @@
     os.environ.get("INTEGREAT_CMS_DEFAULT_REQUEST_TIMEOUT", 10)
 )
 
-###############################
-# Firebase Push Notifications #
-###############################
+
+##############################################################
+# Firebase Push Notifications (Firebase Cloud Messaging FCM) #
+##############################################################
+
+#: API Url
+FCM_URL = "https://fcm.googleapis.com/fcm/send"
```

Comment:
If you're introducing a new setting for the URL (which is a good idea), you should also make use of it and use this setting in https://github.com/digitalfabrik/integreat-cms/blob/a723854ef7d330fe00299a6b89d8f19a515625dd/integreat_cms/api_client_firebase/push_notification_sender.py#L20 instead of the hardcoded class variable.